.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+  - "8"
+cache: yarn
+script: yarn build
```

Comment:
Shouldn't we run `build-data` as well here?

Reply:
Yes we can. Theoretically you should run it before commit.